pymilo/transporters/transporter.py

```diff
@@ -45,11 +44,11 @@ def serialize(self, data, key, model_type):
     def deserialize(self, data, key, model_type):
         """
         Deserialize the data[key] of the given model which type is model_type.
-        
+
         basically in order to fully deserialize a model, we should traverse over all the keys of its serialized data dictionary and
         pass it through the chain of associated transporters to get fully deserialized.
 
-        :param data: the internal data dictionary of the associated json file of the ML model which is generated previously by 
+        :param data: the internal data dictionary of the associated json file of the ML model which is generated previously by
         pymilo export.
```

Comment:
Merge them into one line or justify them by putting pieces of first line into second line.